gabc/details.html

```diff
@@ -308,9 +317,11 @@ <h3 id="braces">Braces above notes</h3>
 
 <h2 id="conclusion">Conclusion and extensions</h2>
 
+<p>Some very advanced features of gabc are documented in GregorioRef-###.pdf (where ### is the version number), please take a look if you cannot find what you need.</p>
+
 <p>If some aspects have been overlooked or if a new written form should appear, many ASCII characters have not yet been used in our notation and remain available, for example, the letters <em>n, p, t</em> and <em>u</em>, both upper- and lower-case.</p>
 
-<p>We hope that we have described here the main aspects that need to be considered for the fast notation of Gregorian chant, and we hope that this notation will be implemented. All questions and remarks are welcome on <a href="mailto:gregorio-devel@gna.org" title="">the gregorio developers mailing list</a>.</p>
+<p>We hope that we have described here the main aspects that need to be considered for the fast notation of Gregorian chant, and we hope that this notation will be implemented. If you think gabc lacks a feature or see a bug, please report on <a href="https://github.com/gregorio-project/gregorio/issues" title="">the issue tracker</a>.</p>
```

Comment:
`and we hope that this notation will be implemented.`  I'm guessing this came from copying a pasting from the original spec sheet, but it's clearly out of place on this page since we're describing what has actually been implemented.  It may also be a good idea to reword most of this section (or at least reorganize it).  I've got to get ready for Vespers now, but I'll think some more about this and make a more constructive suggestion later.